schema/artist/index.js

```diff
@@ -129,6 +130,8 @@ const ArtistType = new GraphQLObjectType({
               auction_lots_count),
             for_sale_artworks: numeral(({ forsale_artworks_count }) =>
               forsale_artworks_count),
+            partner_shows: numeral(({ id }) =>
+              total(`related/shows`, {artist_id: id})),
```

Comment:
Nah; It just requires a function to get a value like you have here. The rest is taken care of: https://github.com/artsy/metaphysics/blob/master/schema/fields/numeral.js

However, your build is failing because of the eslint check (prefer spaces in objects). You can run `npm run lint` locally to check; or I really like http://www.sublimelinter.com/en/latest/ if you're using sublime. (Run `npm test` to run the full test suite and lint that runs in CI)